src/signals/settings/users/containers/Overview/index.js

```diff
@@ -31,11 +32,18 @@ const HeaderButton = styled(Button)`
   }
 `;
 
+const StyledSearchbar = styled(SearchBar)`
+  > button {
+    display: none;
+  }
+`;
+
 export const UsersOverviewContainer = ({ pageSize, userCan }) => {
   const history = useHistory();
   const { pageNum } = useParams();
   const [page, setPage] = useState(1);
-  const { isLoading, users: { list: data }, users } = useFetchUsers({ page, pageSize });
+  const [filters, setFilters] = useState({});
```

Comment:
This should actually be a reducer; having an object in the state will always rerender the component, even when not necessary.